source/Outfit.h

```diff
@@ -14,7 +14,7 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #define OUTFIT_H_
 
 #include "Weapon.h"
-
+#include "Permissions.h"
 #include "Dictionary.h"
```

Comment:
Leave the blank line. `Outfit` inherits from `Weapon` so that header file is included first and has an empty line separating it from the other included headers.
```suggestion

#include "Dictionary.h"
#include "Permissions.h"
```